Neptune.WebMvc/Models/OVTAAreaUploadGisReportJsonResult.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Neptune.Common.DesignByContract;
+using Neptune.EFModels.Entities;
+
+namespace Neptune.WebMvc.Models;
+
+public class OVTAAreaUploadGisReportJsonResult
+{
+    public int StormwaterJurisdictionID { get; set; }
+    public int? NumberOfOVTAAreas { get; set; }
+    public int? NumberOfOVTAAreasToBeUpdated { get; set; }
+    public int? NumberOfOVTAAreasToBeCreated { get; set; }
+    public List<string> Errors { get; set; }
+
+    public static OVTAAreaUploadGisReportJsonResult GetOVTAAreaUploadGisReportFromStaging(NeptuneDbContext dbContext, Person person, ICollection<OnlandVisualTrashAssessmentAreaStaging> ovtaAreaStagings)
+    {
+        var errors = new List<string>();
+        var stormwaterJurisdictions = ovtaAreaStagings.Select(x => x.StormwaterJurisdiction).Distinct().ToList();

```

Comment:
no need to select x.StormwaterJursidiction; use x.StormwaterJurisdictionID and update code below since all it needs is a StormwaterJurisdictionID